build.gradle

```diff
@@ -333,9 +324,8 @@ configure(coreProjects) {
                     }
                     developers {
                         developer {
-                            id = nexusUsername
-                            name = nexusFullName
-                            email = nexusEmail
+                            name = 'Paolo Di Tommaso'
+                            email = 'paolo.ditommaso@gmail.com'
```

Comment:
WHy?! for once it was not hardcoded! 😆

Reply:
Running in CI doesn't have a `~/.gradle/gradle.properties` file to read from.

We could read it from an env var, but won't it always be you anyway? These values essentially represent the project owner in the generated maven pom file, so it probably wouldn't ever be anyone else even if another person or service account was doing the release.

Reply:
Fair enough